charmander/app.py

```diff
@@ -0,0 +1,62 @@
+"""
+Create the application.
+
+"""
+from microcosm.api import create_object_graph
+from microcosm.loaders import load_each, load_from_environ, load_from_json_file
+from microcosm.loaders.compose import load_config_and_secrets
+
+import charmander.postgres  # noqa
+import charmander.routes.example.controller  # noqa
+import charmander.routes.example.crud  # noqa
+import charmander.stores.example_store  # noqa
+from charmander.config import load_default_config
+from microcosm_secretsmanager.loaders.conventions import load_from_secretsmanager
```

Comment:
I'm seeing some weirdly generated import [ordering]. I'm not sure why `isort` things this should be here, but if I put it in the right place it will error. I will debug later.